whois-query/src/main/java/net/ripe/db/whois/query/acl/AccountingIdentifier.java

```diff
@@ -2,21 +2,5 @@
 
 import java.net.InetAddress;
 
-public class AccountingIdentifier {
-
-    final private InetAddress remoteAddress;
-    final private String userName;
-
-    public AccountingIdentifier(InetAddress remoteAddress, String userName) {
-        this.remoteAddress = remoteAddress;
-        this.userName = userName;
-    }
-
-    public String getUserName() {
-        return userName;
-    }
-
-    public InetAddress getRemoteAddress() {
-        return remoteAddress;
-    }
+public record AccountingIdentifier(InetAddress remoteAddress, String userName, String ssoToken) {
```

Comment:
I decided to add the ssoToken coming from the request header in here, as an extra user identification step
There is another option to just keep the userName (as it is) and call the auth service to get the related ssoToken (it is 1 to 1 relation). However, this would allow the user to bypass the counting, putting a wrong email-uuid relation...(we would be ignoring the UUID from the request).
This is needed because in the Auth Index table (which I need to use because of task requirement) we are storing "SSO <UUID>" as value for `Auth:` attribute